class-lab/exercise1/person.js

```diff
@@ -0,0 +1,14 @@
+class Person {
+  constructor(name, age=1) {
+    this.name = name;
+    this.age = age;
+  }
+
+  hello() {
+    console.log("My name is "+ this.name + " and I am " + this.age + " years old");
```

Comment:
To keep code a little cleaner you can use ES6 template strings, like this:
`console.log('My name is {$this.name} and I am ${this.age} years old')`
I cannot use here this symbol ` so I put ' instead